server/public/index.html

```diff
@@ -10,16 +10,16 @@
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
     integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
     crossorigin="anonymous" referrerpolicy="no-referrer" />
-  <link rel="icon" href="favicon.ico">
+  <link rel="icon" href="/server/public/favicon.ico">
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.9.4/css/bulma.min.css"
     integrity="sha512-HqxHUkJM0SYcbvxUw5P60SzdOTy/QVwA1JJrvaXJv4q7lmbDZCmZaqz01UPOaQveoxfYRv1tHozWGPMcuTBuvQ=="
     crossorigin="anonymous" referrerpolicy="no-referrer" />
-  <link rel="stylesheet" href="reset.css">
-  <link rel="stylesheet" href="layout.css">
-  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="/reset.css">
+  <link rel="stylesheet" href="/layout.css">
+  <link rel="stylesheet" href="/styles.css">
 </head>
 <body>
   <div id="root"></div>
-  <script src="main.js"></script>
+  <script src="/server/public/main.js"></script>
```

Comment:
`/` actually translates to `server/public` because you are using static middleware on the back end, so you'll want to change these back to just `/whatever`